CogniteSdk.Types/Common/Items.cs

```diff
@@ -41,4 +41,17 @@ public class ItemsWithCursor<T>
         /// <inheritdoc />
         public override string ToString() => Stringable.ToString<ItemsWithCursor<T>>(this);
     }
+
+    /// <summary>
+    /// Object used to deserialize the result of aggregate (count) requests.
+    /// </summary>
+    public class MatchingCount
```

Comment:
This class is used when unwrapping all the aggregate results, which by default are of the type: ItemsWithoutCursor<MatchingCount>. Do we want this class in a separate file?